pkg/utils/utils_test.go

```diff
@@ -20,32 +20,70 @@ import (
 )
 
 var _ = Describe("Utils", func() {
-	It("must format a short name", func() {
-		chain := FormatChainName("test", "1234")
-		Expect(len(chain)).To(Equal(maxChainLength))
-		Expect(chain).To(Equal("CNI-2bbe0c48b91a7d1b8a6753a8"))
-	})
+	Context("FormatChainName", func() {
```

Comment:
Idiomatic Ginkgo would have this be a `Describe` instead of `Context`.  So that the full test can be read in English like
> Describe [the function] FormatChainName [, ] It must format a short name